tests/test_routes.py

```diff
@@ -493,11 +493,11 @@ def test_cves_filtered_by_version_and_package(self):
         assert add_cves_response.status_code == 200
 
         filtered_cves_response = self.client.get(
-            "/security/cves.json?package=test_package_2&version=testrelease"
+            "/security/cves.json?package=sql&version=testrelease"
         )
 
         assert filtered_cves_response.status_code == 200
-        assert filtered_cves_response.json["total_results"] == 1
+        assert filtered_cves_response.json["total_results"] > 1
```

Comment:
@carkod can I ask what the usefulness is of making this test less specific? We want to know that the correct number of cves are returned with this query, total results being > 1 seems like it could very easily lead to false positives here

Reply:
Well for the purposes of this feature, we have to get more than one result, but sure I can make it more specific.